src/components/style/Modal.css

```diff
@@ -0,0 +1,14 @@
+.modal-overlay{
+  z-index: 2;
+  position: fixed;
+  left: 0;
+  top: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  text-align: center;
+  background-color: rgba(0, 0, 0, 0.5);
```

Comment:
I mentioned it should be a semi transparent white background for the overlay, but this is probably better.